news/2021-03-16-osu!catch-world-cup-2021-staff-and-mapper-applications.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "osu!catch World Cup 2021: Staff and Mapper Applications"
+date: 2021-03-16 14:00:00 +0000
+---
+
+I hope you've been saving up some good fruit puns, because the osu!catch World Cup 2021 is about to begin!
```

Comment:
```suggestion
We hope you've been saving up some good fruit puns, because the osu!catch World Cup 2021 is about to begin, and we're looking for mappers, commentators and referees to help make it happen!
```